### ISSUE TITLE
Update date is not shown

### ISSUE THREAD

**Issue by julezrulez** [w3c/wcag#1088]
On top of the page on https://www.w3.org/WAI/WCAG21/Techniques/ underneath the heading "Techniques for WCAG 2.1" there is an H2 with text "Updated". The date is missing in the following code: `<span class="generate-date"></span>`

Expected behaviour: show the last update date.

**Comment by alastc** [comment]
This has been re-published and the date is there.
